tests/rt.sh

```diff
@@ -494,7 +494,7 @@ fi
 
 INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20211210}
 INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT}/WW3_input_data_20211113
-INPUTDATA_ROOT_BMIC=${INPUTDATA_ROOT_BMIC:-$DISKNM/NEMSfv3gfs/BM_IC-20210717}
+INPUTDATA_ROOT_BMIC=${INPUTDATA_ROOT_BMIC:-$DISKNM/NEMSfv3gfs/BM_IC-20220207}
```

Comment:
I saw data directories cpc,gfs_p7, mom6_da besides gfs_p8a and wav_p7.2 under BM_IC-20220207, are they same data sets as those under BM_IC-20210717? If yes, could we add gfs_p8a and wav_p7.2 under BM_IC-20210717 to avoid the duplications?

Reply:
That works for me.